components/schemas/scoped-variables/RawSource.yml

```diff
@@ -13,12 +13,22 @@ properties:
   details:
     type: object
     required:
-      - string
+      - value
       - blob
     properties:
-      string:
+      value:
         type: string
         description: The value of the source.
```

Comment:
This description can probably be better. Maybe something like "The value of the variable"